src/main/kotlin/no/nav/familie/ef/sak/vedtak/VedtakService.kt

```diff
@@ -100,6 +119,62 @@ class VedtakService(
     fun hentHarAktivtVedtak(behandlingId: UUID, localDate: LocalDate = LocalDate.now()): Boolean {
         return hentVedtak(behandlingId).erVedtakAktivtForDato(localDate)
     }
+
+    @Transactional
+    fun angreSendTilBeslutter(saksbehandling: Saksbehandling) {
+        val vedtak = hentVedtak(behandlingId = saksbehandling.id)
```

Comment:
Vi kan ha tilfelle der 2 ulike saksbehandlere har sendt til totrinnskontroll og lagret selve vedtaket. 
Har en litt følelse at dette skulle vært plassert i `TotrinnskontrollService`, der finnes det idag kode for "hvem som sist sende til totrinnskontroll" i `lagreTotrinnskontrollOgReturnerBehandler`

Reply:
Good catch :) Det blir ikke helt riktig å se på hvem som har lagd vedtaket. Fikser.